src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -95,14 +81,15 @@ def get_packages_to_cover(
   test_target: PythonTestsAdaptor,
   pytest: PyTest,
   test_options: TestOptions,
```

Comment:
Should `get_packagse_to_cover()` move to `coverage.py`?

Reply:
It is only used in this file so it seems a bit odd to stash it elsewhere, even if it is coverage related. 

Possibly I should have a `coverage_util.py` file with all these free functions though.

Reply:
Great idea! Especially now that we have a coverage folder. Could be `coverage/util.py`, for example.